packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -133,6 +133,21 @@ void main() {
             });
       });
 
+      test('init errors', () async {
+        final VideoPlayerController controller = VideoPlayerController.network(
+          'http://testing.com/invalid_url',
+        );
+        try {
+          dynamic error;
+          fakeVideoPlayerPlatform.forceInitError = true;
+          await controller.initialize().catchError((dynamic e) => error = e);
+          final PlatformException platformEx = error;
+          expect(platformEx.code, equals('VideoError'));
+        } finally {
+          fakeVideoPlayerPlatform.forceInitError = false;
```

Comment:
nit: Why the `try/finally` here? Is the `catchError` not really catching the error?

Reply:
Just used to reset the forceInitError flag to ensure if any other error is thrown during testing the flag is reset so other tests can complete without issue.